base/src/main/java/com/smartdevicelink/proxy/rpc/AppCapability.java

```diff
@@ -0,0 +1,53 @@
+package com.smartdevicelink.proxy.rpc;
```

Comment:
@kostyaBoss I checked out your code generator tool: https://github.com/LuxoftSDL/rpc_spec.git

Lines 3273-3296 define the `VideoStreamingCapability` struct.  If you place your `AppCapability` XML below that struct then you should be able to generate it.  Also the XML for `AppCapabilityType` (enum) should be placed above `AppCapability` XML when you do this.

Reply:
@santanamk I've tried this but didn't work for me. Will try rolling back the data manually

Reply:
@kostyaBoss I generated the `AppCapability.java` file using your generator tool.  There are a lot of differences between the generated `AppCapability.java` file and the one you are using.  I have attached the generated `AppCapability.java` here.   It is a .txt file but you can rename the extension to .java.
[AppCapability.txt](https://github.com/smartdevicelink/sdl_java_suite/files/5780086/AppCapability.txt)